upgrade/lib/validation/CASMREL-776-CSM12-NCN-boot-order-backport/scripts/mini-install.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+# Author: Russell Bunch <doomslayer@hpe.com>
+# Permalink to original script: https://github.com/Cray-HPE/node-image-build/blob/8ec558f712bd894792758058fbdabdb6c2addf38/boxes/ncn-common/files/scripts/metal/install.sh
+trap "printf >&2 'Metal Install: [ % -20s ]' 'failed'" ERR TERM HUP INT
+trap "echo 'See logfile at: /var/log/cloud-init-metal.log'" EXIT
+set -e
+
+# Echo that we're the original, but stripped down script.
+echo "Running stripped CSM 1.2 install.sh script, $0"
+# See the original "install.sh" here: https://raw.githubusercontent.com/Cray-HPE/node-image-build/develop/boxes/ncn-common/files/scripts/metal/install.sh?token=AB3JQE4CBRQKCKMZ5UAKHH3BL4264'
```

Comment:
@rustydb can we remove this comment (and the secret token it includes)?

Reply:
Go for it.
It would be nice to file a bug with the guardian for not screaming at hashes, do you know if and how we can do that aside from this PR?